vendor/README.md

```diff
@@ -579,6 +579,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **Vala:** [technosophos/Vala-TMBundle](https://github.com/technosophos/Vala-TMBundle)
 - **Valve Data Format:** [Nixinova/NovaGrammars](https://github.com/Nixinova/NovaGrammars)
 - **Velocity Template Language:** [animecyc/AtomLanguageVelocity](https://github.com/animecyc/AtomLanguageVelocity)
+- **Vento:** [gregory-m/ejs-tmbundle](https://github.com/gregory-m/ejs-tmbundle)
```

Comment:
🤨 I have no idea how you did this. This file is automatically updated when you run the `add-grammar` script.

Reply:
Me neither :)
I will edit it with the correct repository URL.